lib/kernel/src/inet_db.erl

```diff
@@ -1223,7 +1223,9 @@ handle_set_file(Option, Fname, TagTm, TagInfo, ParseFun, From,
 			    {ok, B, _} -> B;
 			    _ -> <<>>
 			end;
-		    _ -> <<>>
+		    _ ->
+			ets:insert(Db, {TagTm, times()}),
```

Comment:
This line should do what the clause for `ParseFun =:= undefined` does:
```
	ets:insert(Db, {TagInfo, undefined}),
	TimeZero = - (?RES_FILE_UPDATE_TM + 1), % Early enough
	ets:insert(Db, {TagTm, TimeZero}),
```
There are use cases where e.g `/etc/resolv.conf` file is written by the Erlang application immediately after system start, and by setting `{TagTm, times()}` a re-read is delayed for quite a while but by setting `{TagTm, TimeZero}` the file will be re-read at first resolver lookup.  Also, `TagInfo` is everywhere else in `inet_db` set in conjunction with `TagTm` so it feels more consistent to set it to `undefined` than to not set it.